test/conformance/api/v1/service_test.go

```diff
@@ -318,8 +318,17 @@ func TestServiceWithTrafficSplit(t *testing.T) {
 	if err := validateDomains(t, clients,
 		names.URL,
 		[]string{expectedFirstRev},
-		[]string{"latest", "current"},
-		[]string{expectedFirstRev, expectedFirstRev}); err != nil {
+		[]trafficExpectedPair{
+			{
+				traffic:  "latest",
+				expected: expectedFirstRev,
+			},
+			{
+				traffic:  "current",
+				expected: expectedFirstRev,
+			},
+		},
+		names.Service); err != nil {
```

Comment:
nit: moving names.Service to be right after clients  and keeping the test expectations last for readability